src/test/java/org/opentripplanner/VariableArgumentsProvider.java

```diff
@@ -0,0 +1,55 @@
+package org.opentripplanner;
```

Comment:
I think maybe a better package would be `org.opentripplanner.test.support` ?

Reply:
I've moved the test utilities into the requested package `org.opentripplanner.test.support`.